### PR TITLE
Add release notes for add-on and solution modules

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

File: content/en/docs/releasenotes/studio-pro/9/9.12.md [hunk 1]
```diff
@@ -28,6 +28,16 @@ This is the preferred [MTS](/releasenotes/studio-pro/lts-mts/#mts) release for a
 * We added the possibility to retry failed workflows. This would restart only the failed activities and put the workflow in progress again. For more information, see [Change Workflow State](/refguide/change-workflow-state/).
 * We introduced a capability to track workflow and user task state changes using state change event microflows. They can be app-wide (in the [App Settings](/refguide/project-settings/)), and/or specific of a certain workflow (overriding the ones defined in **App Settings**). This feature can be used to build an audit system for workflows and user tasks, for example. 
 
+#### Add-on and Solution Modules
+
+We added modularity capability to the platform. [ISV program](https://www.mendix.com/marketplace-vendor-program) members and [private beta partners](/releasenotes/beta-features/) are able to create new types of modules which have IP protection options and offer different distribution mechanisms that allow for easier upgrades management on consumer side. For more information, see [Modules](/refguide/modules/) and [Creating Content](/appstore/creating-content/).
```
Comment (Adam-Dupaski) [inline content/en/docs/releasenotes/studio-pro/9/9.12.md]: Please verify naming of these programs in this doc: https://docs.mendix.com/appstore/creating-content/vendor-program/

Cross-reference should go to that doc.

"Modules" and "Creating Content" should include: "in the *Marketplace Guide*."

Comment (MariaShaposhnikova) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: I see the ISV program there, but not private Beta partners. @mikita-haurykau , where the private Beta partners term is coming from?

Comment (MariaShaposhnikova) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: In my documentation on add-on and solution modules, I mention ISV program and App Services program. Shall I use them in release notes as well?

Comment (mikita-haurykau) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: https://paper.dropbox.com/doc/Release-Readiness-Studio-Pro-9.12-Add-ons-Solutions--Bel65jzhGlGAQwR_qqpkh00nAg-HBlm0anJZpX4Poapnk9J1
Here is a document by @dlhartveld. I took terms from there

Comment (MariaShaposhnikova) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: @mikita-haurykau , I see that David uses "App Services private beta partners" in this document. He provided me with the terms "ISV program" and "App Services program". I see **App Services Vendor** mentioned in https://www.mendix.com/marketplace-vendor-program/. I also see ISV mentioned (but not ISV program specifically) in https://docs.mendix.com/appstore/creating-content/vendor-program/.
Programs mentioned in https://www.mendix.com/marketplace-vendor-program/ are the following:

- App Service Vendor
- Startup Vendor
- Solutions Vendor

Which of them correspond to ISV program?

Comment (MariaShaposhnikova) [reply content/en/docs/releasenotes/studio-pro/9/9.12.md]: @mikita-haurykau , I will stick to a more general term for both: Mendix Vendor program. This is the term that was verified and is used in the documentation. I will change the terms in the release notes and in my other PR.

**Pull request merged**